.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,71 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+name: 'CodeQL'
+
+on:
+  push:
+    branches: [master, development]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [development]
+  schedule:
```

Comment:
I don't think we need to schedule scanning here. Firing the action at each push is enough